### DISCUSSION TITLE
[2358720]Black Myth: Wukong{Black Myth: Wukong+Black Myth: Wukong Deluxe Edition Upgrade}

### DISCUSSION THREAD

**Discussion by 10086pro** [pjy612/SteamManifestCache#837] [Share]
### Steam 游戏ID

2358720

### 密钥列表

```hash
2358721;78d5a284965b2206de191dc4ca99d43c9caf5b5e78b47cf0c366fbbb884190d8
2672611;9f7513ce3953f71d2f994a9a331b862c8a41aeae89b0ff0933336aeef1796220
```

### 清单附件

[2358721_6731736344683312223.zip](https://github.com/user-attachments/files/17054971/2358721_6731736344683312223.zip)

### 备注

Hc-2024@outlook.com

**Comment by pjy612** [comment]
2672611_1995373587248188296 的清单也请提供

**Reply by zoobllzz0001-hue** [reply]
[2358720.zip](https://github.com/user-attachments/files/26765192/2358720.zip)
Here is

**Comment by ReubenGeoffrey** [comment]
Work? 2025?
